plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output.yml

```diff
@@ -26,1464 +26,1552 @@ project:
     dependencies: []
   - name: "external"
     dependencies:
-    - id: "Hackage:Control:mtl:2.2.2"
+    - id: "Hackage::QuickCheck:2.13.1"
```

Comment:
There's also a Windows version of this file. In the past, I've updated it without actually running the test on Windows by manually taking all updates from the non-Windows file and copying them over while maintaining the Windows-specific parts in the expected output.

Reply:
Another alternative would be to simply remove *all* Windows-specific expected results:

```
plugins/package-managers/python/src/funTest/assets/projects/external/example-python-flask-expected-output-windows.yml
plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output-windows.yml
```

And also simplify the test code accordingly, as no one is running tests regularly on Windows anymore anyway.

Reply:
I see, I didn't get that your initial comment was a "change request".

So, If I only change as you describe in comment directly above, it's addressed, right ?

Reply:
Yes, a preceding commit in this PR that removes the Windows-specific test and data would be fine with me.